src/wallet/coincontrol.h

```diff
@@ -60,6 +60,8 @@ class CCoinControl
     int m_max_depth = DEFAULT_MAX_DEPTH;
     //! SigningProvider that has pubkeys and scripts to do spend size estimation for external inputs
     FlatSigningProvider m_external_provider;
+    //! Wallet inputs are signed by an external signer
+    bool m_external_signer = false;
```

Comment:
will be*

But I don't think this is a good name for the flag. It's quite possible external signers might support grinding, and will want a way to optimise fees too?

Reply:
I could call it `m_can_grind_r` and default to `true`? And then for external signers we set it to `false` until we have a way (e.g. via HWI) to communicate this capability. That said, I don't know if anyone will implement it, since Taproot doesn't have this problem.